manager/orchestrator/replicated/restart_test.go

```diff
@@ -803,3 +804,217 @@ func TestOrchestratorRestartWindow(t *testing.T) {
 	assert.Equal(t, observedTask8.DesiredState, api.TaskStateRunning)
 	assert.Equal(t, observedTask8.ServiceAnnotations.Name, "name1")
 }
+
+func TestOrchestratorBackoffValues(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	s := store.NewMemoryStore(nil)
+	assert.NotNil(t, s)
```

Comment:
Like in https://github.com/docker/swarmkit/pull/2466#issuecomment-351217359, this and some other asserts are basically mandatory for the test to make any sense, so change them to `requires`.

It's tempting to use either `assert` or `require` exclusively, but sometimes you want a test to continue running when an assertion fails so that you can see what other failures occur and hopefully better understand what caused the initial test failure. But in cases like this, there's nothing meaningful to be done if the assertion fails; the whole test is useless. You can also change `require` down on 856 and 859, for example.